herbert_core/utilities/general/range_add_border.m

```diff
@@ -0,0 +1,58 @@
+function range=range_add_border(range_in, tol)
+% Add a small border to a range.
+%
+%   >> range=range_add_border(range_in)
+%   >> range=range_add_border(range_in,tol)
+%
+% Input:
+% ------
+%   range_in   Range of data (2xn array)
+%               [u1_min,u2_min,...;u1_max,u2_max,...]
+%   if tol is omitted, the tol assumed to be equal -eps where eps
```

Comment:
let's not be ridiculous.  NaN-s remain Nan-s and let they stay this way. Routine should deal with the situations where one may occur, not with any trouble in the word. 

To is ok.

Reply:
@abuts At this point I'm correcting the English. You can either have "the tol is assumed to equal -eps" or "the tol is assumed to be equal to -eps". I capitalise words when I want to emphasize that they should be added or removed.

For the elseif comment above at line 46, I think it is good that if/elseif/else blocks should have an else so that the logic doesn't accidentally skip without giving notice. NaN was just a possible example. You're welcome either to follow Nick and have the tol<0 branch as a simple else or to keep your tol<o branch and add a full else afterwards with an error. I was just trying to split the difference between the two of you and let you go with his completeness check and your preferred if structure.

Reply:
I've hopefully modified the code within these ideas and have mentioned the changes above, though still not sure these checks add much value. Its developer function, used rarely and mainly internally and thorough check of all possible arguments in the world does not look reasonable though makes no harm (within current usage)